OxfordBodleian/Juel-Jensen/BDLaethg37.xml

```diff
@@ -48,11 +346,15 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                             <supportDesc>
                                 <support>
                                     <material key="parchment"/>
+                                    <p>Hole in the parchment on <locus target="#2r"/>, probably from the parchment making process. Parchment repairs
+                                    on <locus target="#49 #50"/>.</p>
                                 </support>
                                 <extent>
                                     <measure unit="leaf">53</measure>
-                                    <measure unit="leaf" type="blank"></measure>
-                                    <!--<locus target=""/>-->
+                                    <measure unit="leaf" type="blank">5</measure>
+                                    <locus target="#27r #32v #35r #39r #43r"/>
```

Comment:
It's probably not blank, though I don't think this is worth encoding at the same time...

Reply:
This might be worth discussing together. What is a blank leaf? Only those completely blank or those left blank during the production of the main content (on which later something might have been added)?